wordpress-mobile/WordPress-iOS
# Add tracking for day selection in blogging reminders

Adds tracking for the day selection in blogging reminders.

## To test:

IMPORTANT: If notifications are disabled you won't see the tracking calls until you enable them.  The tracking call will be executed when the "Enable notifications" screen refreshes.

When you tap on the "Next" button in the day picker screen, and there's a selection of days, you will see this tracking call (with the selected number of days):

🔵 Tracked: blogging_reminders_scheduled <blog_type: wpcom, days_of_week_count: 2>

OTOH, if you tap on the "Next" button in the day picker screen, and there's no selection of days, you will see this tracking call.

🔵 Tracked: blogging_reminders_cancelled <blog_type: wpcom>

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
